demo/demo-springmvc/springmvc-server/src/main/resources/microservice.yaml

```diff
@@ -47,10 +47,11 @@ servicecomb:
   # can download config center from https://cse-bucket.obs.myhwclouds.com/LocalCSE/Local-CSE-1.0.0.zip to test dynamic config
   config:
     client:
-  #    serverUri: http://127.0.0.1:30113
+      serverUri: http://127.0.0.1:30113
```

Comment:
Because intergartion tests do not install config center, so you can not submit this test case now.